dashboards.go

```diff
@@ -53,45 +52,46 @@ type GraphEvent struct {
 }
 
 type Yaxis struct {
-	Min   *float64 `json:"min,omitempty"`
-	AutoMin *bool
-	Max   *float64 `json:"max,omitempty"`
-	AutoMax * bool
-	Scale *string  `json:"scale,omitempty"`
+	Min     *float64 `json:"min,omitempty"`
+	AutoMin bool     `json:"-"`
```

Comment:
Also FYI, making a field private will also prevent it from being marshalled (although it also prevents consumers from using it, obviously).